Makefile

```diff
@@ -52,6 +54,12 @@ check-pypi-release:
 check-copr-release:
 	PYTHONPATH=$(CURDIR) pytest -m "release_copr" ./tests/release/test_release.py
 
+check-packaging:
+	for test in $(TEST1) $(TEST2) ; do \
+		echo "Checking test: $$test" ; \
+		docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}:/src -v ${PWD}/pytest-container.ini:/src/pytest.ini $(TEST_IMAGE_NAME) make exec-test TEST_TARGET=$$test ; \
```

Comment:
unfortunately this is not the exact thing we need to verify packaging; this would just run unit tests and an integration tests for utilities in a container